app/assets/javascripts/modules/namespaces/components/new-form.js

```diff
@@ -65,40 +65,28 @@ export default {
       name: {
         required,
         format(value) {
-          // extracted from models/namespace.rb
-          const regexp = /^[a-z0-9]+(?:[._-][a-z0-9]+)*$/;
-
-          // required already taking care of this
-          if (value === '') {
-            return true;
-          }
-
-          return regexp.test(value);
-        },
-        available(value) {
           clearTimeout(this.timeout.name);
 
           // required already taking care of this
           if (value === '') {
-            return true;
+            return "";
```

Comment:
Revert this to `true`. We don't need to validate an empty string. That's the responsibility of the required validator that is built-in from the library we are using.